### PR TITLE
done with person service

### PR DESCRIPTION
**Pull request by DoctorFreed**

### CODE REVIEW

File: src/app/person.service.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  createPerson(login: string, password: string): Observable<any> {
+    return this.http
+      .post('http://localhost:8000/person/createPerson', { login, password })
+      .pipe(map((result: any) => result));
+  }
+
+  logining(login: string, password: string): Observable<any> {
+    return this.http
+      .get(
+        `http://localhost:8000/person/login?login=${login}&password=${password}`
```
Comment (usatovaDasha) [inline src/app/person.service.ts:21]: why get request? and you send the password in plain text in the parameters?

File: src/app/registration/registration.component.ts [hunk 2]
```diff
@@ -41,7 +47,14 @@ export class RegistrationComponent implements OnInit {
...
 
   onSubmit() {
-    alert(JSON.stringify(this.regForm.value));
+    this.personService
+      .createPerson(
+        this.regForm.controls['login'].value,
+        this.regForm.controls['password'].value
+      )
+      .subscribe((result) => {
+        this.token = result.token;
```
Comment (usatovaDasha) [inline src/app/registration/registration.component.ts:56]: use localstorage for save token

Comment (DoctorFreed) [reply src/app/registration/registration.component.ts:56]: yep i planned this, now fix)

File: src/app/person.service.ts (hunk 1 above)
Comment (DoctorFreed) [reply src/app/person.service.ts:21]: because in backend have get root to logining and get has no body, only query

Comment (usatovaDasha) [reply src/app/person.service.ts:21]: so change to a post request in order to correctly and covertly pass the parameters)

Comment (DoctorFreed) [reply src/app/person.service.ts:21]: okay, i change backend

**Pull request merged**